newscoop/install/Resources/sql/upgrade/4.4.x/2015.06.15/tables.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE  `Aliases` CHANGE  `IdPublication`  `IdPublication` INT( 10 ) UNSIGNED NULL DEFAULT  '0'
```

Comment:
why not null but 0 instead? wont that throw an error because there wont be ever publication with id 0 ?